blueprints/controllers/ikea_e1812/ikea_e1812.yaml

```
@@ -130,34 +106,34 @@ variables:
   integration: !input integration
   button_long_loop: !input button_long_loop
   button_long_max_loop_repeats: !input button_long_max_loop_repeats
-  button_double_press: !input button_double_press
-  helper_last_controller_event: !input helper_last_controller_event
-  helper_double_press_delay: !input helper_double_press_delay
   helper_debounce_delay: !input helper_debounce_delay
   # integration id used to select items in the action mapping
   integration_id: '{{ integration | lower }}'
   # adjusted debounce delay so that the resulting double press delay is exactly as specified by the user when running the action, taking also account of debouncing
   # make sure it never goes below the minimum double press delay
-  adjusted_double_press_delay: '{{ [helper_double_press_delay - helper_debounce_delay, 100] | max }}'
   # mapping between actions and integrations
   actions_mapping:
     deconz:
       button_short: ['1002']
       button_long: ['1001']
       button_release: ['1003']
+      button_double: ['1004']
     zha:
       button_short: ['on']
       button_long: [move_with_on_off_0_83]
       button_release: [stop]
+      button_double: ['off']   # UNTESTED
```

```suggestion
      button_double: ['off']
```
Remove the untested label since we received positive feedback from a ZHA user.